website/layouts/api.jsx

```diff
@@ -9,42 +9,40 @@ import order from '../data/api-navigation.js'
 
 const MDXProvider = createMdxProvider({ product: 'consul' })
 
-function ApiDocsLayoutWrapper(pageMeta) {
```

Comment:
In each of the changes files under `layouts`, I've followed [the `next-mdx-enhanced` migration instructions for 13.0.0](https://github.com/hashicorp/nextjs-scripts/releases/tag/13.0.0) and have used [Nomad's layout files](https://github.com/hashicorp/nomad/tree/master/website/layouts) as a reference.